README.md

```diff
@@ -1,10 +1,88 @@
 # CARTA Frontend
+The carta-frontend is used in conjunction with the [carta-backend](https://github.com/CARTAvis/carta-backend), and for general use a pre-built production carta-frontend is usually installed alongside the carta-backend automatically. 
+However, a user may wish to install the carta-frontend component separately in order to test beta versions, for example, or to develop it.
+
+## Pre-built production packages
+The simplest way to install the carta-frontend component is to use the pre-built production packages from NPM (any OS with Node.js installed), our PPA (Debian packages for Ubuntu), our RPM repository (for RHEL based systems), or Homebrew (for MacOS).
+
+### NPM
+
+Pre-built production carta-frontends can be obtained from the [npmjs.com carta-frontend repository](https://www.npmjs.com/package/carta-frontend?activeTab=explore).
+
+If you have Node.js and `npm` already installed on your system, a production carta-frontend can be installed using the following command:
+
+```
+npm i carta-frontend
+```
+The files will be installed to your `node-modules` directory.
+Specific carta-frontend versions can be installed by appending `@` and the version number. For example,
+```
+npm i carta-frontend@1.4.0
+npm i carta-frontend@2.0.0-beta.0
+```
+The available versions are listed in the [npmjs.com carta-frontend versions tab](https://www.npmjs.com/package/carta-frontend?activeTab=versions).
+
+Alternatively, the pre-built production carta-frontend can be downloaded directly without npm. For example,
+```
+wget https://registry.npmjs.org/carta-frontend/-/carta-frontend-2.0.0.tgz
+```
+The `tgz` extracts to `package/build`. 
```

Comment:
Is this a relative path? Relative to what, `node-modules`? This should be more explicit.